src/elements/content-sidebar/activity-feed/comment/__tests__/Comment-test.js

```diff
@@ -173,7 +173,7 @@ describe('elements/content-sidebar/ActivityFeed/comment/Comment', () => {
 
         const instance = wrapper.instance();
 
-        expect(wrapper.find('CommentMenu').length).toEqual(2);
+        expect(wrapper.find('CommentMenu').length).toEqual(1);
```

Comment:
I'm not sure why, but when we were doing `injectIntl()` it was creating two `CommentMenu` components in these tests. Now it's back to one.
<img width="1368" alt="Screen Shot 2019-06-18 at 10 29 13 PM" src="https://user-images.githubusercontent.com/1280912/59739799-df059200-921a-11e9-942b-360aeefa311e.png">